lib/tasks/plugin.rake

```diff
@@ -146,3 +146,32 @@ task 'plugin:qunit', [:plugin, :timeout] do |t, args|
 
   sh cmd
 end
+
+desc 'run all migrations of a plugin'
+namespace 'plugin:migrate' do
+  def list_migrations(plugin_name)
+    plugin_root = File.join(Rails.root, "plugins", plugin_name)
+    migrations_root = File.join(plugin_root, "db", "migrate", "*.rb")
```

Comment:
I think we might need `post_migrate` in there as well [e.g.](https://github.com/discourse/discourse/tree/master/plugins/poll/db/post_migrate)

Reply:
Can you confirm this is correct:
- one migration in january
- one post migratrion in february
- one migration in march

the post migration has to happen before march for up and after for down ?

Reply:
I think for this purpose we should just run them in order of date. That's how it works for self-hosters, and in development. The only time it makes a difference is when you set RUN_POST_DEPLOYMENT_MIGRATIONS=0

Reply:
oh so I can just mix them all together, cool

Reply:
Ok That should do it now :+1: